ts/src/binance.ts

```diff
@@ -976,15 +973,6 @@ export default class binance extends Exchange {
                         'exchangeInfo': 4, // Weight(IP): 20 => cost = 0.2 * 20 = 4
                         'avgPrice': 0.4,
                     },
-                    'put': {
-                        'userDataStream': 0.4,
-                    },
-                    'post': {
-                        'userDataStream': 0.4,
-                    },
-                    'delete': {
-                        'userDataStream': 0.4,
-                    },
```

Comment:
userDataStream might need to be private: https://github.com/binance/binance-spot-api-docs/blob/master/rest-api.md#user-data-stream-endpoints
also the code suggests so, see this:
https://github.com/ccxt/ccxt/blob/bb2675e88774562f637fb25a77607d1010dc4823/ts/src/binance.ts#L11186-L11199

all other market types (sapi, fapi etc..) also have `userDataStream` endpoint under private